presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -492,11 +494,7 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .function(TRY_CAST)
                 .function(DECIMAL_AVERAGE_AGGREGATION)
                 .function(DECIMAL_SUM_AGGREGATION)
-                .functions(DECIMAL_CEILING_FUNCTIONS)
-                .function(DECIMAL_FLOOR_FUNCTION)
-                .function(DECIMAL_MOD_FUNCTION)
-                .functions(DECIMAL_ROUND_FUNCTIONS)
-                .function(DECIMAL_TRUNCATE_FUNCTION);
+                .function(DECIMAL_MOD_FUNCTION);
```

Comment:
don't add function at the end because this causes conflicts. Please leave TRY_CAST or DECIMAL_SUM_AGGREGATION or whatever there was last.